File: OCH_Tao/ABC/B/021.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const N = parseInt(input[0], 10);
const [A, B] = (input[1].split(" ")).map(x => parseInt(x, 10));
const K = parseInt(input[2], 10);
const P = (input[3].split(" ")).map(x => parseInt(x, 10));
P.push(A);
P.push(B);
const X = [...new Set(P)];
if (P.length !== X.length) {
  console.log("NO");
} else {
  console.log("YES");
}

File: OCH_Tao/ABC/B/021.py
N = int(input())
A,B = map(int,input().split())
K = int(input())
P = list(map(int,input().split()))
P.append(A)
P.append(B)
if len(P) != len(set(P)):
  print("NO")
else:
  print("YES")